runtimes/runtimes/standalone.ts

```diff
@@ -89,9 +90,40 @@ if (checkAWSConfigFile()) {
     process.env.AWS_SDK_LOAD_CONFIG = '1'
 }
 
-process.on('uncaughtExceptionMonitor', err => {
-    console.error('Uncaught Exception:', err.message)
-})
+// Set up crash monitoring to emit telemetry before process exits
+function setupCrashMonitoring(telemetryEmitter?: (metric: MetricEvent) => void) {
+    function getTopStackFrames(err: Error): string | undefined {
+        if (!err.stack) {
+            return undefined
+        }
+
+        const stackLines = err.stack.split('\n')
+        const topStackFrames = stackLines.slice(0, 10).join('\n')
+        return topStackFrames
+    }
+
+    process.on('uncaughtExceptionMonitor', (err, origin) => {
+        console.error('Uncaught Exception:', err.message)
+
+        if (telemetryEmitter) {
+            try {
+                telemetryEmitter({
```

Comment:
For this metric to be emitted from toolkits, I believe it needs to be added [here](https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json). The fields should also be defined as types there, but I think there are ones you can reuse. 

This will also require bumping the version of the telemetry package in vscode [here](https://github.com/aws/aws-toolkit-vscode/blob/8e85476708e0d2e9467cc7afe13a42661de78f1d/package.json#L44) once that change is in (as well as other IDES).

Reply:
Can you update the PR Description with a sample of the latest changes?